_data/navigation.yml

```diff
@@ -305,6 +305,15 @@ guides:
     children:
       - title: Custom Webhook Stages
         url: /guides/operator/custom-webhook-stages/
+      - title: Runbooks
```

Comment:
Any particular reason to keep "Runbooks" as an intermediate item, rather than moving its children directly under "Operator How-Tos"?

Reply:
I put it here per your #1035. But since #988, there are now some conflicts. I'm going to close this PR and then revisit the organization of the ops guides.